src/main/scala/Column.scala

```diff
@@ -452,6 +463,57 @@ case class Column private (
       s"${xmlSchema}time" -> processTime _
     )
 
+  val datatypeFormatValidation = Map(
+    "http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral" -> regexpValidation _,
+    "http://www.w3.org/1999/02/22-rdf-syntax-ns#HTML" -> regexpValidation _,
```

Comment:
Use an `xmlSchemaPrefix` variable to avoid making mistakes in here.

Reply:
Done